files/templates/mbta/login/messages/messages_pt.properties

```diff
@@ -42,12 +42,12 @@ invalidPasswordRegexPatternMessage=Senha: não corresponde ao (s) padrão (s) re
 invalidPasswordHistoryMessage=Senha: não deve ser igual a nenhuma das últimas {0} senhas.
 invalidPasswordGenericMessage=Senha: a nova senha não corresponde às políticas de senha.
 
-secondfactor.userNotFound=O usuário que enviou o e-mail de segundo fator não foi encontrado
+secondFactor.userNotFound=O usuário que enviou o e-mail de segundo fator não foi encontrado
 secondFactor.sendEmailError=Desculpe, houve um problema ao enviar o e-mail do segundo fator
 secondFactor.validateCodeTitle=Insira o código de verificação
-secondfactor.code=Insira o código de verificação única do seu e-mail.
-secondfactor.badeMailCode=Código único incorreto
-secondfactor.userNoEmail=O usuário não tem e-mail
+secondFactor.code=Insira o código de verificação única do seu e-mail.
+secondFactor.badeMailCode=Código único incorreto
```

Comment:
question: this disagrees with `messages_es`, which has `secondFactor.badEmailCode`. which one is right?

Reply:
It is `badEmailCode`, corrected in https://github.com/mbta/keycloak-deploy/commit/9f6315ec968af070bffadddb5650032f7af38f55